freeipa/src/main/java/com/sequenceiq/freeipa/service/user/UserService.java

```diff
@@ -84,19 +138,28 @@ private void addUsers(FreeIpaClient freeIpaClient, Set<User> users) throws FreeI
 
             LOGGER.debug("adding user {}", username);
 
-            RPCResponse<com.sequenceiq.freeipa.client.model.User> userAdd = freeIpaClient.userAdd(
-                    username, user.getFirstName(), user.getLastName(), generateRandomPassword());
-
-            LOGGER.debug("Success: {}", userAdd.getResult());
+            try {
+                com.sequenceiq.freeipa.client.model.User userAdd = freeIpaClient.userAdd(
+                        username, user.getFirstName(), user.getLastName(), generateRandomPassword());
```

Comment:
Presumably we'll communicate this generated password to the user somehow?

Reply:
users need a password so that we can generate a keytab. This password here is not saved. We have another API that the user can use to set their password. That password will then only be known by the user and not by the control plane